### ISSUE TITLE
XGBClassifier' object has no attribute 'n_jobs'

### ISSUE THREAD

**Issue by ShaniCohen** [dmlc/xgboost#3148]
`xgboost` version used: 0.6
python 3.6

got the following error:

  File "C:\Anaconda\lib\site-packages\xgboost\sklearn.py", line 536, in predict_proba
    test_dmatrix = DMatrix(data, missing=self.missing, nthread=self.n_jobs)

AttributeError: 'XGBClassifier' object has no attribute 'n_jobs'

how can I solve the problem?

**Comment by SePTimO7** [comment]
for the sklearn wrapper, param "n_estimators" is used

**Comment by vasulakkaraju** [comment]
facing the same issue,any fix for this?

**Comment by zhanghuiCinderella** [comment]
i have the same problem ,did you solve it ?

**Comment by zhanghuiCinderella** [comment]
i figure out that this problem happend because i used different xgboost version    , xgboost 0.7  XGBClassifier has attribute  'n_jobs' ,xgboost 0.6 and 0.4 has not

**Comment by comlaterra** [comment]
I'm using `0.71` and I get the same error. Where did you find out the version that have the attribute? I'll check out the changenotes.

**Comment by evants** [comment]
I had this error as well.  I resolved it by ensuring the xgboost version that I used to train my model was the same version that was being used when I deserialize the model and classified with it.
In my case the model was built with 0.6a2 and I was using 0.71 when the error occurred.  I suspect this is because the n_jobs is written to the classifier in 0.6a2, but 0.71 (and 0.72) expects the classifier to call this n_threads.
